src/backend/aspen/app/views/sample.py

```diff
@@ -265,10 +266,9 @@ def samples():
         for cansee in user.group.can_see
         if cansee.data_type == DataType.PRIVATE_IDENTIFIERS
     }
-
-    # load the samples.
     samples: Sequence[Sample] = (
         g.db_session.query(Sample)
+        .outerjoin(Sample.uploaded_pathogen_genome, Sample.sequencing_reads_collection)
```

Comment:
we have an outerjoin here, but also a joinedload below -- are these redundant? Why both?

Reply:
i needed to add that to get the table aliases for the desc calls later on, but now i decided to handle this all on the frontend so i removed these changes. sqlalchemy is weird sometimes.